lib/rucio/core/subscription.py

```diff
@@ -44,32 +43,33 @@
 
 
 @transactional_session
-def add_subscription(name, account, filter, replication_rules, comments, lifetime, retroactive, dry_run, priority=3, session=None):
+def add_subscription(name, account, filter, replication_rules, comments, lifetime, retroactive, dry_run, priority=3, session=None, logger=logging.log):
```

Comment:
Why is the logger added here? It doesn't seem to be used anywhere in the function.